cmake/modules/FindSUNDIALS.cmake

```diff
@@ -24,15 +24,14 @@
 #   SUNDIALS_LIB_SER
 #   SUNDIALS_INCLUDE_DIR
 #
-# Note that, unlike most other packages, SUNDIALS headers are typically
-# installed in multiple directories; for example, a typical system installation
-# may distribute the headers in the following way:
+# Note that sundials headers are typically installed in several directories,
+# e.g.,
 #
-# /usr/include/ida/ida.h
-# /usr/include/kinsol/kinsol.h
-# /usr/include/sundials/sundials_nvector.h
+# /prefix/ida/ida.h
```

Comment:
to avoid confusion, here and below i would write 
```
prefix/include/ida/ida.h
```
because libraries are located in 
```
prefix/lib/libsundials_cvode.a
```
etc.

i think `prefix` suggests that it's installation prefix.

Reply:
i guess on your system they must be in `/usr/lib` or `/usr/lib64`